### ISSUE TITLE
Note for subproperties of Date - double-checking...

### ISSUE THREAD

**Issue by tombaker** [dcmi/usage#64]
In correcting the latest ISO 15836-2, I find that a number of decisions we made about the usage notes for Date and its subproperties arguably supersede each other in subtle ways.

In August, for example, we decided that the following usage note should be used for all subproperties of Date:

    Note 1 to entry: Recommended practice is to describe
    the date, time, or period as recommended for the
    property Date, of which this property is a
    refinement.

However, we subsequently decided to use "subproperty of" consistently instead of "refines".  On November 2, then, we decided that the usage comment for Date should read:

    Recommended practice is to express the date,
    date/time, or period of time according to ... etc

Applying the decisions in order, I come up with the following usage note to be added to all subproperties of Date:

    Note 1 to entry: Recommended practice is to describe
    the date, date/time, or period of time as recommended
    for the property Date, of which this is a
    subproperty.

The subproperties of Date to which this would apply are:

* http://purl.org/dc/terms/available
* http://purl.org/dc/terms/created
* http://purl.org/dc/terms/dateAccepted
* http://purl.org/dc/terms/dateCopyrighted
* http://purl.org/dc/terms/dateSubmitted
* http://purl.org/dc/terms/issued
* http://purl.org/dc/terms/modified
* http://purl.org/dc/terms/valid

If you disagree with my interpretation, please let me know as soon as possible.  In particular, I am having some doubts about whether we really intended to say that _all_ of the date subproperties can have a "period of time" as a value, which is what the notes say.

**Comment by tombaker** [comment]
AMENDED COMMENT FOR DATE SUBPROPERTIES

    Note 1 to entry: Recommended practice is to describe
    the date, date/time, or period of time as recommended
    for the property Date, of which this is a
    subproperty.

**Comment by kcoyle** [comment]
I'm not worried about including 'period of time' for all of the date subproperties. I think people will already have in mind what kind of date they have and will ignore what isn't relevant to them.

I do note, however, the inconsistency of naming - that some do not have "date" in their names which makes them ambiguous as terms. The concept of a date on which something is valid is already a stretch, but calling it simply "valid" makes it worse. Obviously this is not for this iteration but I would hope that we would do better given a chance.

**Comment by tombaker** [comment]
@kcoyle The naming style is inconsistent because `valid` was approved in the [original batch of "qualifiers"](http://dublincore.org/documents/2000/07/11/dcmes-qualifiers/) in July 2000.  At the time, there was a notion that qualifiers such as "valid" were like adjectives: i.e., "valid" (adj) "date" (noun).

By 2002, when `dateAccepted` was approved, `dateApproved` was seen as an element refinement or, in today's terminology, a sub-property.

In the Usage Board, we took note of the inconsistency but it seemed to be a non-starter to create new URIs simply to make the URIs more consistent.  We changed the label of `valid` to `Date Valid' but left the URI untouched.
